hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -70,7 +70,7 @@
                          change of state (this is pure, but handled in Python
                          to simplify verification).
 
-    send_lc_escalation   React to lc_escalate_en_i going high
+    send_err_escalation  React to bad internal fsm state
```

Comment:
Both of these descriptions might be true :-) Probably "React to an injected error" ?